app.js

```diff
@@ -151,8 +154,9 @@ app.post('/auth/', function(request, response){
 app.get('/' + apiPath.base + '/test', function(request, response){
 	var responseBody = {};
 
-	responseBody.message = "Endpoint hit successfully";
+  blockchainSetup.setup();
```

Comment:
Setup on deploy then I've slipped it into a test enpoint for now also so that we can retry without redeploying (if testing on bluemix)